src/Components/Header/index.jsx

```diff
@@ -35,25 +40,24 @@ const Header = () => {
       <div className="nav__block">
         <div className='nav__block__links'>
           <ul>
-            <li><NavLink to='/'>Home</NavLink></li>
-            <li><NavLink to='/sing-in'>Sing in</NavLink></li>
+            <li><Link to='/'>Home</Link></li>
+            <li><Link to='/sing-in'>Sing in</Link></li>
           </ul>
         </div>
         <Route path='/' exact>
           <div className='nav__block__buttons'>
         <div>
-              <span className='buttons'>
-            <Checkbox onClick={onView} type="checkbox" id='check1' />
-            <label htmlFor='check1'>View Mode</label>
+          <span className='buttons'>
+          <Checkbox onClick={onViewHandler} type="checkbox" id='check1' />
```

Comment:
Once the onView Mode is enabled you should not have a chance to edit cards. However, if you double-click to any card and follow to specific card page - you'll be able to edit the card. Fix it.